packages/xchain-cosmos/src/client.ts

```diff
@@ -214,14 +214,14 @@ class Client implements CosmosClient, XChainClient {
 
   // Need to be updated
   getFees = async (): Promise<Fees> => {
```

Comment:
do we still need `getFees` to be async ?

Reply:
^ Same reason as mentioned in https://github.com/xchainjs/xchainjs-lib/pull/70#discussion_r517481630